clients/src/main/java/org/apache/kafka/clients/admin/ExpireDelegationTokenOptions.java

```diff
@@ -25,10 +25,19 @@
  * The API of this class is evolving, see {@link Admin} for details.
  */
 @InterfaceStability.Evolving
-public class ExpireDelegationTokenOptions extends AbstractOptions<ExpireDelegationTokenOptions> {
+public class ExpireDelegationTokenOptions
```

Comment:
Could you please revert this unrelated change?

Reply:
done.